src/Employer/Employer.Web/Orchestrators/Part2/ApplicationProcessOrchestrator.cs

```diff
@@ -67,20 +67,11 @@ public async Task<OrchestratorResponse> PostApplicationProcessEditModelAsync(App
         {
             var vacancy = await Utility.GetAuthorisedVacancyForEditAsync(_client, m, RouteNames.ApplicationProcess_Post);
 
-            if (_featureToggler.IsFeatureEnabled(FeatureNames.AllowThroughFaaApplicationMethod))
```

Comment:
Shall we just remove the references and dependency of the `IFeature` feature toggle from this class and it's using statement.

Reply:
Yep. Get rid of using statement and ctor injection